tests/testthat/test-drive_update.R

```diff
@@ -22,23 +22,35 @@ test_that("drive_update() updates file", {
   skip_on_appveyor()
   skip_on_travis()
   skip_if_offline()
-  x <- drive_find(nm_("foo"))
-  y <- drive_update(system.file("DESCRIPTION"), nm_("foo"))
-  expect_identical(y$id, x$id)
+
+  updatee <- drive_find(nm_("update_me"))
+  tmp <- tempfile()
+  now <- as.character(Sys.time())
+  writeLines(now, tmp)
+
+  out <- drive_update(tmp, updatee)
+  expect_identical(out$id, updatee$id)
+  drive_download(updatee, tmp, overwrite = TRUE)
+  now_out <- readLines(tmp)
+  expect_identical(now, now_out)
 })
```

Comment:
I made this a more genuine test of whether we have, in fact, updated anything at all!